### PR TITLE
9.0 [MIG] partner_sector migration

### PR DESCRIPTION
**Pull request by cubells**

### CODE REVIEW

File: partner_sector/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# © 2016 Tecnativa S.L. - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Partner Sector",
+    "summary": "Add partner sectors",
+    "version": "9.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "website": "http://www.tecnativa.com",
+    "author": "Tecnativa S.L., Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline partner_sector/__openerp__.py]: Only `Tecnativa`, without `S.L.`

File: partner_sector/i18n/partner_sector.pot [hunk 2]
```diff
@@ -0,0 +1,106 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline partner_sector/i18n/partner_sector.pot:1]: Remove this file

File: partner_sector/models/res_partner_sector.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                                ondelete='restrict')
+    child_ids = fields.One2many(comodel_name='res.partner.sector',
+                                inverse_name='parent_id',
+                                string="Children")
+    parent_left = fields.Integer('Parent Left', select=True)
+    parent_right = fields.Integer('Parent Right', select=True)
+
+    @api.multi
+    def name_get(self):
+        def get_names(cat):
```
Comment (pedrobaeza) [inline partner_sector/models/res_partner_sector.py:27]: Add a test for this name_get method

File: partner_sector/models/res_partner_sector.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+class ResPartnerSector(models.Model):
+    _name = 'res.partner.sector'
+    _order = "parent_left"
+    _parent_order = "name"
+    _parent_store = True
+    _description = "Sector"
+
+    name = fields.Char(required=True, translate=True)
+    parent_id = fields.Many2one(comodel_name='res.partner.sector',
```
Comment (pedrobaeza) [inline partner_sector/models/res_partner_sector.py]: Add a check on recursion.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
You have to remove also .gitkeep file and please squash all the "OCA transbot" and the "en.po removal" commits in one.

**Comment by pedrobaeza** [comment]
cc @Tecnativa

**Review by rafaelbn — APPROVED** [review]
Tested functionally in runbot

**Comment by pedrobaeza** [comment]
Remove .gitkeep file.

**Comment by pedrobaeza** [comment]
@cubells, please attend remaining comments

**Comment by cubells** [comment]
@pedrobaeza I thinl all changes are done.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**